src/lib/Group.ts

```diff
@@ -225,4 +226,8 @@ export class Group<
       throw e
     }
   }
+
+  async consentState(): Promise<'allowed' | 'denied' | 'unknown'> {
```

Comment:
Hey I am wondering whether this is the final state for the group `consentState`? 

I want to display which members of a group have accepted the group invite or not. AFAIK this can only be done on the client for now? 

If this function returned the state of each individual member then it would be easier to do directly from the network. 

Is this something that might be implemented in the future?

Reply:
Hmmm not sure I understand the question? Are you looking for a way that every person in the group can see if someone else has approved or denied the group? Inherently the personal private preferences are just that private and personal. So no one else in the group can decrypt if your blocked that person or group only you can.